packages/plugin-react/src/index.ts

```diff
@@ -16,27 +15,16 @@ export interface Options {
   include?: string | RegExp | Array<string | RegExp>
   exclude?: string | RegExp | Array<string | RegExp>
   /**
-   * Enable `react-refresh` integration. Vite disables this in prod env or build mode.
-   * @default true
-   */
-  fastRefresh?: boolean
-  /**
-   * Set this to `"automatic"` to use [vite-react-jsx](https://github.com/alloc/vite-react-jsx).
+   * @deprecated
```

Comment:
I think it would be good to add a comment here about why the option is deprecated.